README.adoc

```diff
@@ -131,20 +127,47 @@ InventoryLivenessCheck.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[]
 ----
 
+A health Check for liveness allows third party services to determine if the application is running. 
+This means that if this procedure fails the application can be discarded (terminated, shutdown).
```

Comment:
This means that if this procedure fails the application can be discarded (terminated, shutdown).
-->
If this procedure fails, the application can be stopped.